sdks/python/apache_beam/runners/interactive/testing/integration/tests/screen_diff_test.py

```diff
@@ -28,7 +28,7 @@
 @pytest.mark.timeout(300)
 class DataFramesTest(BaseTestCase):
   def __init__(self, *args, **kwargs):
-    kwargs['golden_size'] = (1024, 10000)
+    # kwargs['golden_size'] = (1024, 10000)
```

Comment:
Is there context on this?

Reply:
Ah sorry, meant to go back and outright remove this. There was some strangeness around this kwarg when I bumped the pytest version, with it making it all the way up to the base test class despite the fact that it should have been popped off the kwarg dict prior. But this value is _also_ the default golden_size we set if that kwarg isn't passed in the base class, so we can remove it.